tests/test_sigmaJs.py

```diff
@@ -128,7 +128,7 @@ def test_create_node_with_negation_dict_for_sigmaJs():
         'borderColor': '#ff0000',
         'color': '#00ff00',
         'id': 1,
-        'label': 'and',
+        'label': 'NAND',
```

Comment:
I don't like using NAND or NOR in the OVAL visualization. OVAL specification doesn't define NAND and NOR. OVAL doesn't use "normal" binary logic, it defines the operations over different set. Saying that, people can be confused by meaning of "NAND" or "NOR" here. Also, they will not find these operations in their OVAL definitions.

I'm not saying it's bad, but I'm not sold on that either. Could you please elaborate on why you have chosen this solution? I'm missing any rationale in the PR description.

Have you considered having a special node for the negate operation? An OVAL criterion which involves negation would be represented by 2 nodes in the graph - child node containing the OVAL operation (AND, OR, etc.) and parent node containing the negation?

Reply:
Good point. I used binary logic to display negation. And what about using an exclamation mark to show negation and create some description? I think creating a special node is a good idea, but at the moment it will be need big change. In the future it will be good to create. You can creat issue.

Reply:
Or I can use propositional logic. And use &not; `&not;` to display the negation. I think it will be better. What do you think? @jan-cerny